Exception on orientation

Hi, 
thanks for your nice library.
There is a problem which occurs by calling update after orientation changes
`Caused by: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.remove(ArrayList.java:503)
        at com.visualizer.amplitude.AudioRecordView.handleNewFFT(AudioRecordView.kt:125)
        at com.visualizer.amplitude.AudioRecordView.update(AudioRecordView.kt:77)`

--- a/audio/src/main/java/com/visualizer/amplitude/AudioRecordView.kt
+++ b/audio/src/main/java/com/visualizer/amplitude/AudioRecordView.kt
@@ -5,6 +5,7 @@ import android.graphics.Canvas
 import android.graphics.Color
 import android.graphics.Paint
 import android.util.AttributeSet
+import android.util.Log
 import android.view.View
 import java.util.*
 
@@ -120,8 +121,7 @@ class AudioRecordView : View {
 
         val chunkHorizontalScale = chunkWidth + chunkSpace
         val maxChunkCount = width / chunkHorizontalScale
-
-        if (chunkHeights.size >= maxChunkCount) {
+        if (chunkHeights.isNotEmpty() && chunkHeights.size >= maxChunkCount) {
             chunkHeights.removeAt(0)
         } else {
             usageWidth += chunkHorizontalScale

